tdrs-frontend/src/components/Reports/Reports.jsx

```diff
@@ -125,6 +125,24 @@ function Reports() {
     setTouched((currentForm) => ({ ...currentForm, stt: true }))
   }
 
+  const constructYearOptions = () => {
+    const years = []
+    const today = new Date(Date.now())
+
+    const fiscalYear =
+      today.getMonth() > 8 ? today.getFullYear() + 1 : today.getFullYear()
```

Comment:
This ternary will check if the month is past September. If so it sets the Fiscal year to the next calender year. Otherwise it uses the current calendar year.